src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -637,26 +692,32 @@ private static BlockEntry<Block> createCleanroomFilter(IFilterType filterType) {
     }
 
     protected static BlockEntry<ActiveBlock> createActiveCasing(String name, String baseModelPath) {
-        return REGISTRATE.block(name, ActiveBlock::new)
+        var builder = REGISTRATE.block(name, ActiveBlock::new)
                 .initialProperties(() -> Blocks.IRON_BLOCK)
                 .addLayer(() -> RenderType::cutoutMipped)
                 .blockstate(GTModels.createActiveModel(GTCEu.id(baseModelPath)))
-                .tag(GTToolType.WRENCH.harvestTags.get(0), BlockTags.MINEABLE_WITH_PICKAXE)
-                .item(BlockItem::new)
+                .tag(GTToolType.WRENCH.harvestTags.get(0));
+        if (!ConfigHolder.INSTANCE.machines.requireGTToolsForBlocks) {
+            builder.tag(BlockTags.MINEABLE_WITH_PICKAXE);
+        }
+        return builder.item(BlockItem::new)
                 .model((ctx, prov) -> prov.withExistingParent(prov.name(ctx), GTCEu.id(baseModelPath)))
                 .build()
                 .register();
     }
 
     private static BlockEntry<ActiveBlock> createFireboxCasing(BoilerFireboxType type) {
-        BlockEntry<ActiveBlock> block = REGISTRATE
+        var builder = REGISTRATE
                 .block("%s_casing".formatted(type.name()), ActiveBlock::new)
                 .initialProperties(() -> Blocks.IRON_BLOCK)
                 .properties(p -> p.isValidSpawn((state, level, pos, ent) -> false))
                 .addLayer(() -> RenderType::cutoutMipped)
                 .blockstate(GTModels.createFireboxModel("%s_casing".formatted(type.name()), type))
-                .tag(GTToolType.WRENCH.harvestTags.get(0), BlockTags.MINEABLE_WITH_PICKAXE)
-                .item(BlockItem::new)
+                .tag(GTToolType.WRENCH.harvestTags.get(0));
+        if (!ConfigHolder.INSTANCE.machines.requireGTToolsForBlocks) {
+            builder.tag(BlockTags.MINEABLE_WITH_PICKAXE);
+        }
```

Comment:
this won't work for removing the pickaxe tags as it's only applied when you run datagen.